File: Programming-with-Java/Even.java
package com.cmru.java;

// Java program to print even numbers 1 to 100

public class Even {
    public void header(){
        System.out.println("-------------------------------------------------------------------------------");
        System.out.println("|                   JAVA PROGRAM BY KRITIK AGARWAL 19BBTCS067                 |");
        System.out.println("-------------------------------------------------------------------------------");
    }
    public static void main(String[] args){
        Even e = new Even();
        e.header();
        for(int i=1; i<=100; i++){
            if(i%2==0)
                System.out.println(i);
        }
    }
}

File: B.Tech @ CMR University/Programming-with-C/EVEN.c
// EVEN NUMBERS
/*
#include<stdio.h>

int main()
{	
	for(int n=1;n<=20;n++)
	{
		if(n%2==0)
			printf("\n %d \n",n);
	}
	return 0;
}


#include<stdio.h>
int main()
{	
	int n=1;
	while(n<=20)
	{
		if(n%2==0)
			printf("\n %d \n",n);
		n++;
	}
	return 0;
}
*/	
#include<stdio.h>
int main()
{	
	int n=1;
	do
	{
		if(n%2==0)
			printf("\n %d \n",n);
		n++;
	}while(n<=20);
	return 0;
}

